lesson_17/adding_products.py

```diff
@@ -0,0 +1,36 @@
+import psycopg2
+
+connection = psycopg2.connect(
+    dbname="pcf",
+    user="billy",
+    password="Billy",
+    host="127.0.0.1",
+    port="5432"
+)
+
+cur = connection.cursor()
+pcf_table = """CREATE TABLE product
```

Comment:
if I try to repeat this operation - I will get an Error here.. something like `Table already exists`

Reply:
It means my table will contain only one line..
you had to separate it on modules or use `create if not exists..`